tabbededitor.pro

```diff
@@ -31,12 +31,14 @@ DEFINES += TABBEDEDITOR_LIBRARY
 
 SOURCES += \
     tabbededitorplugin.cpp \
-    tabsforeditorswidget.cpp
+    tabsforeditorswidget.cpp \
+    qcontexttabwidget.cpp
```

Comment:
Rename the file, and please keep `SOURCES` and `HEADERS` lists sorted alphabetically.